python/lsst/pipe/tasks/insertFakes.py

```diff
@@ -142,6 +149,11 @@ def _add_fake_sources(exposure, objects, calibFluxRadius=12.0, logger=None):
             exposure[subBox].mask.array |= bitmask
 
 
+@deprecated(
```

Comment:
Probably better to not add a warning to this function, since it is called once for every fake source in the catalog. Plus, it was a private function that is only used here.